test/extended/prometheus/prometheus_builds.go

```diff
@@ -102,18 +102,18 @@ type prometheusResponseData struct {
 	Result     model.Vector `json:"result"`
 }
 
-type metricTest struct {
-	labels map[string]string
-	// we are not more precise (greater than only, or equal only) becauses the extended build tests
+type MetricTest struct {
```

Comment:
Those utils should just move to `test/extended/utils` and build tests should move to builds and use utils then. But that can be done in a followup PR.